.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea
 vendor
+*.swp
+.DS_Store
```

Comment:
little tip, you can add that in your global gitignore
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore